README.md

```diff
@@ -53,15 +32,15 @@ You can simply start the frontend:
 npm run serve
 ```
 
-ATTENTION: Lunie requires a backend. This is currently not yet public.
+The front end will run on its own, however if you want to feed it data to interact with, please read on below.
```

Comment:
@faboweb how do you think we should handle this?

Reply:
What are you suggesting?